environments/staging/inventory.ini

```diff
@@ -88,7 +88,6 @@ celery1
 
 [pillowtop:children]
 # Amazon EC2
-pillow0
```

Comment:
I think you need to make this change to `inventory.ini.j2` and also remove the machine from terraform.

Reply:
I am not yet delete the VM only removing it from inventory